concourse/scripts/pack.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -l
+
+set -exo pipefail
+
+gp_home=""
+file_name=""
+
+while getopts "p:f:h" opt; do
+    case $opt in
+        h)
+            echo "Usage: arg.sh -p gp_home -f tar_file_path"
+            exit 0
+            ;;
+        p)
+            gp_home="$OPTARG"
+            ;;
+        f)
+            file_name="$OPTARG"
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG"
+            exit 1
+            ;;
+        :)
+            echo "Option -$OPTARG requires an argument."
+            exit 1
+            ;;
+    esac
+done
+
+if [ -z "$gp_home" ] || [ -z "$file_name" ]; then
+    echo "-p and -f option should be required"
+    exit 1
+fi
+
+source $gp_home/greenplum_path.sh
+
+cd $GPHOME
+echo 'cp -r lib share $GPHOME || exit 1'> install_gpdb_component
+chmod a+x install_gpdb_component
+
+tar -czf "$file_name" \
+    "lib/postgresql/ip4r.so" \
+    "share/postgresql/extension/ip4r.control" \
+    "share/postgresql/extension/ip4r--2.4.sql" \
```

Comment:
Since we never made a release before, so we don't have to ship those upgrade scripts. Just add the latest, 2.4, should be fine.